src/scenes/Dashboard/components/TopActions/index.js

```diff
@@ -33,24 +38,32 @@ class TopActions extends Component {
     super(props);
 
     this.onSortOptionSelected = this.onSortOptionSelected.bind(this);
+    this.onCreateDialogClose = this.onCreateDialogClose.bind(this);
+    this.onCreateDialogOpen = this.onCreateDialogOpen.bind(this);
   }
 
   onSortOptionSelected(event) {
     this.props.sortOrderChanged(event.target.value);
   }
 
+  onCreateDialogClose() {
```

Comment:
use fat arrow function. Alex showed me this recently and seems handy so we dont have to remember to bind it.
`onCreateDialogClose = () => {
}`